example/scripts/lock-stack.js

```diff
@@ -42,6 +42,9 @@ function updateLock(lockFile, deployment) {
 // node lock-stack.js (true|false) deployment-name
 // true to lock, false to unlock
 updateLock(process.argv[2], process.argv[3]).catch((e) => {
-  console.log(e);
   process.exitCode = 100;
+  if (! e.code === 'ConditionalCheckFailedException') {
```

Comment:
Is there a reason you didn't do `(e.code !== ....)`?

Reply:
That's largely because it would require an else conditonal to display the trace on an unexpected failure - setting a default with a except case  felt more concise/readable than an if/else , even with the negated condition.